crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -78,12 +73,16 @@ impl Settlement {
                 score,
             );
         }
-        if score < promised_solution.score() {
-            return Err(Error::ScoreMismatch {
-                expected: promised_solution.score(),
-                got: score,
-            });
-        }
+        // temporarily disabled due to inconsistencies with rounding of surplus
```

Comment:
Cool, let's disable it by removing it completely and use version control to put it back when the rounding is fixed. Temporarily commented out code is a smell.